sagemaker-training-compiler/huggingface/tensorflow_multiple_gpu_multiple_node/language-modeling.ipynb

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
<p>Add more detailed computation and comparison between the two billing times? It seems like you're only looking at optimized training time here, not native training time. Relates back to Miyoung's comment that this example does not use a native estimator.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>